nj/conf.py

```diff
@@ -0,0 +1,15 @@
+"""conf"""
```

Comment:
Fair enough!! Totally understand. Hey, I'm all about making it work! I absolutely understand if you'd need more time to make these proper changes. I'll await those changes prior to merging, though 😄